node-js-basics-101/node_modules/node-static/lib/node-static.js

```diff
@@ -0,0 +1,393 @@
+var fs     = require('fs')
+  , events = require('events')
+  , buffer = require('buffer')
+  , http   = require('http')
+  , url    = require('url')
```

Comment:
__[jshint]__
* Misleading line break before ','; readers may interpret this as an expression boundary.

<!-- Comment by [SideCI](https://sideci.com) -->